docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
```

Comment:
Please add this document to `.gitattributes` so it will not be delivered with the extension package. (The lines in that file are sorted as well.)